internal/cli/dbusers/create.go

```diff
@@ -83,7 +83,7 @@ func (opts *CreateOpts) isOIDCSet() bool {
 }
 
 func (opts *CreateOpts) isExternal() bool {
-	return opts.isX509Set() || opts.isAWSIAMSet() || opts.isLDAPSet() || opts.isOIDCSet()
```

Comment:
some latest change was done to the dbusers, blocking $external value on OIDC creation, so this is fixing a bug on the OIDC DBUsers creation. cc @tibulca